packages/xl-docx-exporter/src/docx/defaultSchema/blocks.ts

```diff
@@ -133,15 +134,26 @@ export const docxBlockMappingForDefaultSchema: BlockMapping<
   },
   // TODO
   codeBlock: (block, exporter) => {
+    const textContent = (block.content as StyledText<any>[])[0]?.text || "";
+
     return new Paragraph({
-      // ...blockPropsToStyles(block.props, exporter.options.colors),
-      style: "Codeblock",
-      children: exporter.transformInlineContent(block.content),
-      // children: [
-      //   new TextRun({
-      //     text: block..type + " not implemented",
-      //   }),
-      // ],
+      run: {
```

Comment:
it might be nicer to define this style externally (in styles.xml). Optional if you want to do this for this PR, but it's good to know the file exists and how it works when you start working on other exporter related PRs

Reply:
Moved everything except `shading` - it seems to apply differently from `<w:shd/>` in `styles.xml` (doesn't result in uniform background). I might be missing something here though, so let me know if you have any insight into that